plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableRefundDataStore.php

```diff
@@ -159,8 +161,17 @@ public function update_order_meta( &$refund ) {
 
 		$props_to_update = $this->get_props_to_update( $refund, $meta_key_to_props );
 		foreach ( $props_to_update as $meta_key => $prop ) {
-			$value = $refund->{"get_$prop"}( 'edit' );
-			$refund->update_meta_data( $meta_key, $value );
+			$meta_object        = new WC_Meta_Data();
```

Comment:
wdyt about adding a new method to datastore meta that gets metadata by key without having to pass a meta object?

Reply:
We are kind of doing that below on line 167. I guess we could change the order so that we look for existing meta before generating a new one.

Or do you mean making it so that the `update_meta` and `add_meta` methods work without passing in a full meta object?

Reply:
> Do you mean making it so that the update_meta and add_meta methods work without passing in a full meta object?

Yeah this, there have been asks from the community to have a lightweight way to update meta, but now that I think more about it, it probably needs more consideration. I think we are good to go here as that should be handled in a separate PR.